modules/virt-about-cpu-and-memory-quota-namespace.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-cdi-for-namespace-resourcequota.adoc
+
+[id="virt-about-cpu-and-memory-quota-namespace_{context}"]
+= About CPU and memory quotas in a namespace
+
+A resource quota, defined by the `ResourceQuota` object imposes restrictions on
+a namespace that limit the total amount of compute resources that can be
+consumed by resources within that namespace.
+
+The `CDIConfig` object defines the user configuration for CDI. The CPU and
```

Comment:
s/for CDI/for the Containerized Data Importer (CDI).

Since this module could appear outside of this assembly, you need to write out the acronym here, too.